ReportService/Operations/DataExporters/Dependencies/B2BDbStructureChecker.cs

```diff
@@ -28,19 +26,15 @@ public class B2BDbStructureChecker : DBStructureChecker
 				                        AND Object_ID = Object_ID('{ExportInstanceTableName}'))  
                 SELECT 1
                 ELSE SELECT 0";
-        #endregion
-
-
-        #region overrides
 
         public override async Task<bool> CheckIfDbStructureExists(DbConnection connection, IReportTaskRunContext taskContext)
         {
-            var result = await connection.QueryFirstOrDefaultAsync<int>(new CommandDefinition(DbStructureCheckQuery,
-                     new { taskId = taskContext.TaskId }, commandTimeout: DbTimeOut,
-                     cancellationToken: taskContext.CancelSource.Token));
+            Guard.Against.Null(taskContext, nameof(taskContext));
+
+            int result = await connection.QueryFirstOrDefaultAsync<int>(new CommandDefinition(DbStructureCheckQuery, new { taskId = taskContext.TaskId }, 
```

Comment:
is wide strings better than short?

Reply:
For me now it really improves readability in most of cases (easier to see where one line/declaration ends and next starts). But it is more matter of taste and size of monitor

Reply:
1. eyes do 10 times less work if each parameter starts in one column
2. it's hard to find somth in one long string
3. don't need to scroll left to right
4. use ident so you never ever lost your string ending

ex:
Method(
    param1,
    param2,
    anotherOne);

nextString